Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/transform/audit/PatientDiscoveryTransforms.java

```
@@ -1201,6 +1130,59 @@ public String getPatientDiscoveryMessageCommunityId(PRPAIN201305UV02 requestMess
     }
 
     /**
+     * @return
+     */
+    protected String getLocalHCID() {
+        String hcid = StringUtils.EMPTY;
+        try {
+            hcid = PropertyAccessor.getInstance().getProperty(NhincConstants.HOME_COMMUNITY_ID_PROPERTY);
+        } catch (PropertyAccessException e) {
+            LOG.error("Could not retrieve local HCID from gateway.properties", e);
+        }
+        return hcid;
+    }
+
+    protected String getHCIDFromReceiver(PRPAIN201305UV02 requestMessage) {
```

I typically don't like empty catch blocks, but Im fine with this approach. Updating now.